taskcluster/app_services_taskgraph/transforms/module_build.py

```diff
@@ -12,6 +12,25 @@
 transforms = TransformSequence()
 
 
+@transforms.add
+def release_upload_symbols(config, tasks):
+    for task in tasks:
+        if config.params["tasks_for"] == u"github-release":
+            if task["attributes"]["buildconfig"]["uploadSymbols"]:
```

Comment:
Nit: these 2 `if` statements can be combined.

Reply:
Good catch, done!